hash/snes.xml

```diff
@@ -61618,6 +62163,84 @@ to ensure nothing has been touched in the Retro Quest cart production in 2013/20
 		</part>
 	</software>
 
+	<software name="sureinst">
+		<description>Sure Instinct (NTSC, v1.0.2)</description>
+		<year>2021</year>
+		<publisher>Benjamin Schulte</publisher>
+		<info name="version" value="v1.0.2" />
+		<part name="cart" interface="snes_cart">
+			<feature name="slot" value="lorom" />
+			<dataarea name="rom" size="524288">
+				<rom name="SureInstinct NTSC (v1.0.2).sfc" size="524288" crc="9d4e6d2f" sha1="34ed5af8cd1f384adcbecbcc7fa5ee977ee30014" offset="0x000000" />
+			</dataarea>
+		</part>
+	</software>
```

Comment:
Please try to use mnemonic short names for all the clones of this, too.

Reply:
all fixed